src/rpc/net.cpp

```diff
@@ -193,6 +214,8 @@ static UniValue getpeerinfo(const JSONRPCRequest& request)
         obj.pushKV("subver", stats.cleanSubVer);
         obj.pushKV("inbound", stats.fInbound);
         obj.pushKV("addnode", stats.m_manual_connection);
+        obj.pushKV("conn_type", stats.m_conn_type);
+        obj.pushKV("conn_type_name", std::get<1>(CONN_TYPES.at(stats.m_conn_type)));
```

Comment:
This is not safe. If you shuffle the enum then this returns bad values.

Reply:
I'm confused, why would the enum be shuffled? If the enum is not stable, it shouldn't be exposed via the API.

Reply:
Dropped this field.